src/main/java/com/example/rentalSystem/global/auth/jwt/JwtAuthenticationFilter.java

```diff
@@ -0,0 +1,58 @@
+package com.example.rentalSystem.global.auth.jwt;
+
+import com.example.rentalSystem.global.auth.jwt.component.JwtTokenProvider;
+import com.example.rentalSystem.global.exception.custom.CustomException;
+import com.example.rentalSystem.global.response.ApiResponse;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.util.StringUtils;
+import org.springframework.web.filter.GenericFilterBean;
+
+@RequiredArgsConstructor
+@Slf4j
+public class JwtAuthenticationFilter extends GenericFilterBean {
```

Comment:
```suggestion
public class JwtAuthenticationFilter extends OncePerRequestFilter {
```
요청시 한번만 실행되게 하도록 GenericFilterBean을 상속받은 OncePerRequestFilter를 상속받는 것이 더 좋은 것 같습니다

Reply:
찾아본 결과 둘의 차이점은 알았고 궁금한 점이 생겼습니다.
저번 프로젝트할 때 어쩐지 jwt 부분의 결과 로그가 2번씩 찍혔었는 데 그 이유가 무엇이였을 까요. 새로운 api를 호출한 것도 아니였을 텐데.